server/routes/api/place.py

```diff
@@ -380,22 +384,60 @@ def get_related_place(dcid,
                                 is_per_capita=is_per_capita)
 
 
-@cache.memoize(timeout=3600 * 24)  # Cache for one day.
+@cache.memoize(timeout=3600 * 24)
+def get_place_cohort(cohort):
+    """Get all the place dcids for a cohort set."""
+    return dc.get_property_values([cohort], 'member')[cohort]
+
+
 @bp.route('/similar/<stats_var>/<path:dcid>')
 def api_similar_places(stats_var, dcid):
     """
-    Get the similar places for a given place by stats var within the same place.
+    Get the similar places for a given place.
+
+    If the place is a USA city or county, the similar places are randomly
+    selected from curated cohort. Otherwise, the similar places are by stats
+    var within the same place.
     """
+    # Choose city from US city cohort.
+    match = re.match(r'geoId/\d{7}', dcid)
+    if match:
+        city_cohort = get_place_cohort(CITY_COHORT)
+        random.shuffle(city_cohort)
+        result = []
+        for city in city_cohort:
+            if city != dcid:
+                result.append(city)
+            if len(result) == 5:
+                return Response(json.dumps(result),
+                                200,
+                                mimetype='application/json')
+
+    match = re.match(r'geoId/\d{5}', dcid)
+    if match:
+        county_cohort = get_place_cohort(COUNTY_COHORT)
+        random.shuffle(county_cohort)
+        for county in county_cohort:
+            if county != dcid:
+                result.append(county)
+            if len(result) == 5:
+                return Response(json.dumps(result),
+                                200,
+                                mimetype='application/json')
+
     parents = parent_places(dcid)[dcid]
```

Comment:
should we return blank here, as a way to suppress comparison charts for non-US places?

Reply:
It's a bit hard to know if a place is within United States. Shall we consolidate the disabling in client side with the function to check if a parent place is US.

Reply:
ok that sgtm

Reply:
sure, sgtm